pkg/graveler/committed/meta_range_manager.go

```diff
@@ -50,21 +50,9 @@ func (m *metaRangeManager) Exists(ctx context.Context, ns graveler.StorageNamesp
 // GetValue finds the matching graveler.ValueRecord in the MetaRange with the rangeID
 func (m *metaRangeManager) GetValue(ctx context.Context, ns graveler.StorageNamespace, id graveler.MetaRangeID, key graveler.Key) (*graveler.ValueRecord, error) {
```

Comment:
I don't think this is being used anywhere

Reply:
Let's remove in a separate PR?